DscResources/MSFT_WindowsPackageCab/MSFT_WindowsPackageCab.psm1

```diff
@@ -62,12 +62,13 @@ function Get-TargetResource
 
     $getWindowsPackageParams = @{
         PackageName = $Name
-        Online = $true
+        Online =  [switch]::Present
     }
 
     if ($PSBoundParameters.ContainsKey('LogPath'))
     {
         $getWindowsPackageParams['LogPath'] = $LogPath
+        $windowsPackageCab['LogPath'] = $LogPath
```

Comment:
This line is not needed as LogPath is added in the declaration of $windowsPackageCab above.